core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -4075,6 +4122,7 @@ class KafkaApisTest extends Logging {
         assertEquals(authorizedTopic, metadataResponseTopic.name())
       }
     }
+    kafkaApis.close()
```

Comment:
should this be in a finally as well?

Reply:
No, this one is special. If the 1st one throws exception before it closes, we'll close it in the `AfterEach`.